utils/src/test/kotlin/FileMatcherTest.kt

```diff
@@ -50,6 +50,18 @@ class FileMatcherTest : WordSpec({
                 defaultMatcher.matches(it) shouldBe true
             }
         }
+
+        "match commonly used license file paths in lower-case" {
+            COMMONLY_USED_LICENSE_FILE_NAMES.map { it }.forAll {
```

Comment:
Either do `it.toLowerCase()` or omit `map`.

Reply:
Done.

Reply:
Done.